src/braket/experimental/autoqasm/instructions/gates.py

```diff
@@ -20,8 +20,8 @@
 
 from braket.circuits.free_parameter_expression import FreeParameterExpression
 
+from ..types import QubitIdentifierType
```

Comment:
[nit] we've been using the full path for these, i.e. `braket.experimental.autoqasm.types` rather than `..types`.

Reply:
I think pycharm matches whatever is used in the file and probably picked up on the relative import right below. I'll update both

Reply:
Getting import errors when I switch this to absolute imports, maybe that's why the line below is using a relative import? Not sure that this is worth prioritizing before the Program refactor, which will likely alter file structure